core/src/nucleus/mod.rs

```diff
@@ -403,24 +404,24 @@ fn reduce_execute_zome_function(
     );
 }
 
-/// Reduce ValidateEntry Action
-/// Validate an Entry by calling its validation function
+/// Reduce ValidatePair Action
+/// Validate a Pair by calling its validation function
```

Comment:
Since we are removing pair, maybe rename this `ValidateEntryHeader`? (and remove all remaining references to pair)